core/controllers/base.py

```diff
@@ -523,6 +526,12 @@ def delete(self, *args):  # pylint: disable=unused-argument
         """
         raise self.PageNotFoundException
 
+    def head(self, *args, **kwargs):
+        """Method to handle HEAD requests. It returns only
+        the headers of GET request.
```

Comment:
Is this done automatically? Since you seem to call the get as usual.

Reply:
@vojtechjelinek Yeah it is done automatically. Should I mention it in the code comment?

Reply:
Mentioned it in the code comment